packages/gatsby-image/src/index.js

```diff
@@ -332,24 +430,24 @@ class Image extends React.Component {
 
           {/* Show the blurry base64 image. */}
           {image.base64 && (
-            <Img src={image.base64} {...placeholderImageProps} />
+            <picture>
+              {generateBase64Sources(imageVariants)}
+              <Img src={image.base64} {...placeholderImageProps} />
```

Comment:
From [what I can find](https://stackoverflow.com/questions/25396605/when-targeting-a-picture-element-in-css-should-we-use-img-or-picture-selector) on StackOverflow, people generally recommend applying styles to the `<img>` instead of the `<picture>`, partly to avoid the issue that you raised, and partly because it's ultimately the `<img>` tag that gets rendered, not the `<picture>`. We can think of it as metadata to the browser instead of an element that we need to target with styling, so I think keeping the styles on `<img>` still makes sense here.

And since `<picture>` elements are [inline](https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements#Elements), it won't have any impact on layout like a `<div>` would.